src/components/views/settings/feedback.tsx

```diff
@@ -71,7 +70,6 @@ ENS Details:
   exposure permission: ${permissions.exposure.status},
   notifications permission: ${permissions.notifications.status},
   authorised: ${isAuthorised ? 'yes' : 'no'},
-  amount of closeness cases: ${contacts?.length || 0}
```

Comment:
Should we keep this as `contacts?.length > 0`? Not sure about the label

Reply:
@floridemai I thought about that but judging by the wording of the issue, that's something you'd need to negotiate and get agreement from their third party stakeholders:

> business stakeholders agree that its technical value for debugging is not commensurate with the privacy loss

You'd need to make the case that revealing whether a device has any contacts or not is small privacy wise and has large technical value